extensions/sql-database-projects/src/dialogs/addDatabaseReferenceDialog.ts

```diff
@@ -651,12 +675,11 @@ export class AddDatabaseReferenceDialog {
 		return label;
 	}
 
-	private createInputBox(ariaLabel: string, enabled: boolean, required: boolean): azdataType.InputBoxComponent {
+	private createInputBox(ariaLabel: string, enabled: boolean): azdataType.InputBoxComponent {
 		const inputBox = this.view!.modelBuilder.inputBox().withProps({
 			ariaLabel: ariaLabel,
 			enabled: enabled,
-			width: cssStyles.addDatabaseReferenceInputboxWidth,
-			required: required
```

Comment:
can you check if this is needed for screenreaders to announce the input boxes as required when they're navigated to?

Reply:
Yeah, screen reader didn't announce it as required. Added it back.